_includes/comparison.html

```diff
@@ -59,7 +59,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://istio.io/docs/tasks/traffic-management/ingress/" target="_blank">Envoy / Own Concept</a></td>
         <td>any</td>
         <td></td>
-        <td><a href="https://www.consul.io/docs/connect/gateways/ingress-gateway" target="_blank">Envoy</a> and <a href="https://www.consul.io/docs/k8s/connect/ambassador" target="_blank">Ambassador in Kubernetes</a></td>
+        <td><a href="https://www.consul.io/docs/connect/gateways/ingress-gateway" target="_blank">Envoy</a> and <a href="https://www.consul.io/docs/k8s/connect/ambassador" target="_blank">Ambassador in Kubernetes</a>, or any</td>
         <td>any</td>
```

Comment:
I am not sure I understand. The first link seems to explain that any Ingress Gateway can be managed by Consul using the Envoy proxy. The second link shows how to use the Ambassador Ingress Gateway with Consul. So maybe it should say "any with the Envoy proxy, e.g. Ambassador" and give the links for Envoy and Ambassador?

Reply:
Its possible to use any ingress when using transparent proxy. An Envoy sidecar will be injected alongside the ingress to facilitate communication with services in the mesh.

Consul also provides its own out-of-the-box ingress gateway built on Envoy. Ambassador and Traefik (in the next release) also support native integrations with Consul which do not require an additional sidecar to be deployed.

I'll revert the change and will look to address it in a subsequent PR.